### ISSUE TITLE
Advise users that script in Git for Windows needs GnuWin 'zip' command installed.

### ISSUE THREAD

**Issue by samkatakouzinos** [KuriKai/dhmp#11]
TLDR; People who git clone this repository and wish to run the scripts need to prepare their Git for Windows environment with the zip command.

The script calls for zip.

Git for Windows does not have the zip command by default.
It needs to be downloaded and installed correctly for the bash script to run correctly.  

I followed instructions at https://ranxing.wordpress.com/2016/12/13/add-zip-into-git-bash-on-windows/ which allowed the bash script to run without error. 

GnuWin zip can be obtained from https://sourceforge.net/projects/gnuwin32/files/zip/3.0/zip-3.0-bin.zip/download
GnuWin bzip2 (required by zip) can be downloaded from https://sourceforge.net/projects/gnuwin32/files/bzip2/1.0.5/bzip2-1.0.5-bin.zip/download

**Comment by KuriKai** [comment]
Are you able to update the readme with instructions on how to do it and request a pull? I don't have windows so don't know how to do it.
